code/lib/physical_models.py

```diff
@@ -450,7 +450,7 @@ def calc_CF_wind(hour, reg_ind, turbine, m, n, merraData, rasterData):
 
     # Load MERRA data, increase its resolution, and fit it to the extent
     w50m_h = resizem(merraData["W50M"][:, :, hour], m, n)
-    w50m_h = w50m_h[reg_ind]
+    w50m_h = w50m_h[tuple(reg_ind)]
```

Comment:
why?

Reply:
Using a list of arrays instead of a tuple will become depreciated in the future.
Raises a future_warning